charts/pact-broker/templates/deployment.yaml

```diff
@@ -254,5 +250,17 @@ spec:
               path: /diagnostic/status/heartbeat
               port: http
           {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+      {{- toYaml .Values.nodeSelector | nindent 8 }}
```

Comment:
do you need to set the `nindent` to 4?

Reply:
Yes, in deployment we need this ident (https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#more-practical-use-cases), another example could you see at nginx-ingress helm chart (https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/controller-deployment.yaml#L199).

Reply:
This needs to be `.Values.broker.nodeSelector`.

Reply:
```suggestion
      {{- toYaml .Values.broker.nodeSelector | nindent 8 }}
```